wdl/src/main/scala/wdl/If.scala

```diff
@@ -69,6 +62,14 @@ object If {
     }
     val possiblyNeededNestedOginPorts: Map[String, OutputPort] = possiblyNeededNestedOgins map { case (name: String, ogin: OuterGraphInputNode) => name -> ogin.singleOutputPort }
 
+    val ifConditionExpression = WdlWomExpression(ifBlock.condition, ifBlock)
+    val ifConditionGraphInputExpressionValidation = WdlWomExpression.toAnonymousExpressionNode(
+      WomIdentifier("conditional"), ifConditionExpression, localLookup ++ possiblyNeededNestedOginPorts, Map.empty, preserveIndexForOuterLookups, ifBlock, PlainAnonymousExpressionNode.apply)
```

Comment:
This line isn't just a copy/paste. Note the `..., localLookup ++ possiblyNeededNestedOginPorts, Map.empty, ...`

Reply:
I see the `possiblyNeededNestedOginPorts ` being added to the `localLookup` but I'm just wondering why the `outerLookup` has been replaced with `Map.empty` ?

Reply:
oh nevermind I think I got it

Reply:
I'm actually tempted to remove the outer lookup entirely and have callers bundle in "possible needed ogins" as local inputs. 

Since that's my go-to fix to all these "duplicate ogin created" bugs, being proactive about removing the other cases might not be such a bad idea. Not in this PR though...

Reply:
Yeah as long as it doesn't mess up with `preserveIndexForOuterLookups` I agree

Reply:
yeah this would have been a great candidate for split commits but I think I get it as much as I'm going to anytime soon...